grails-app/assets/javascripts/streama/templates/modal--media-detail.tpl.htm

```diff
@@ -86,7 +98,7 @@ <h4>{{'VIDEO.TRAILER' | translate}}</h4>
        </div>
      </div>
      <div class="col-sm-4">
-       <img ng-show="media.poster_path" ng-src="https://image.tmdb.org/t/p/w300/{{media.poster_path}}"/>
+       <img ng-if="media.poster_path" ng-show="media.poster_path" ng-src="https://image.tmdb.org/t/p/w300/{{media.poster_path}}"/>
```

Comment:
Either the ng-if here or the ng-show can be removed. Not necessary to have both with the same condition